.github/workflows/deploy-npm.yml

```diff
@@ -103,13 +103,13 @@ jobs:
               ]
             }
 
-      - name: Send message to Slack channel for v1.1
-        id: slackv1
+      - name: Send message to Slack channel for Service Worker
```

Comment:
What benefit does it have by altering the Slack message sequence?

Reply:
I want to separate the v1.1 and v3 flows, so removing them later will be easy.